### DISCUSSION TITLE
Is it possible to set the recurring period length?

### DISCUSSION THREAD

**Discussion by JessYang525** [jonas-hoebenreich/wc-donation-platform#75] [Q&A]
Can the length of the recurring period be set for recurring donations?

**Comment by jonas-hoebenreich** [comment]
Yes that's possible :)

https://www.wc-donation.com/documentation/integrations/recurring-donations-with-woocommerce-subscriptions/
https://www.wc-donation.com/documentation/integrations/setting-up-variable-recurring-donations-with-woocommerce-subscriptions/
